app/src/pages/api/eligibility-screener.ts

```diff
@@ -0,0 +1,124 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
```

Comment:
Notes for reviewers: When the user clicks "submit" on `/review`, we make a client-side call to this next.js API endpoint. This API endpoint prepares the data to be sent on to an external api (specifically this one: https://github.com/navapbc/wic-mt-demo-project-mock-api). This API handles building the body, using `fetch()` to connect to the mock API, and calling the mock API's endpoint.